post/views.py

```
@@ -0,0 +1,260 @@
+                {"detail": "[username, password] fields missing."},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        ### 2 ###
+        try:
+            author = User.objects.get(username=username)
+            if not author.check_password(password):
+                return Response(
+                    {"detail": "Password is incorrect."},
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)